btr-api/src/btr_api/utils/redact.py

```diff
@@ -104,6 +106,19 @@ def redact_field(field, redact_type):
                 rv += ' '
             rv += word[0:1] + '***'
         redacted_field = rv
+    elif redact_type == RedactionType.REDACT_MONONYM_LN:
+        rv = ''
```

Comment:
I tested this function locally, and for `Little Red Riding Hood` I get: `Little Red R*** H***`
is that what we want ?

if we want to redact every word you can use following:

```
   temp = []
    words = field.split()
    for word in words:
        temp.append(word[0:1] + '***')
    rv = ' '.join(temp)
    return rv
```

Reply:
We aren't using REDACT_MONONYM_LN I was working on it and realized this case and when I dug into it we aren't using the modifiers as we have to show the full name. The other use potential problem with the function is when for instance someone is 'Jr.' or 'the III' because these go after the last name

Reply:
With the above stated I have updated the function to properly only redact the last word so Little Red Riding Hood becomes Little Red Riding H*** and Queen Elizabeth II becomes Queen Elizabeth I***